src/main/java/com/scalar/cassy/command/AbstractCommand.java

```diff
@@ -33,12 +33,11 @@
       paramLabel = "DATA_DIR",
       description = "A data directory to take backups from or restore backup to")
   protected String dataDir;
-
   @CommandLine.Option(
       names = {"--store-type"},
       required = true,
       paramLabel = "STORE_TYPE",
-      description = "The store type [AWS_S3, AZURE_STORAGE_BLOB]")
+      description = "The store type [AWS_S3, FILE_SYSTEM, AZURE_STORAGE_BLOB]")
```

Comment:
Maybe `REMOTE_FILESYSTEM` for consistency ? I will think about the name once again.

Reply:
Sure, I will change to `REMOTE_FILESYSTEM`. I will make sure the naming is also consistent in the readme and variable names.

Reply:
Please wait. Let me think about it first.

Reply:
Sorry for late. Let's use `REMOTE_FILESYSTEM` for now.